pkg/registry/core/event/strategy.go

```diff
@@ -121,7 +121,7 @@ func ToSelectableFields(event *api.Event) fields.Set {
 		"source":                         source,
 		"type":                           event.Type,
 	}
-	return generic.MergeFieldsSets(objectMetaFieldsSet, specificFieldsSet)
+	return generic.MergeFieldsSets(specificFieldsSet, objectMetaFieldsSet)
```

Comment:
Why this helps?
It's because we resize the map once instead of twice? How much it helps?

Reply:
SG - thanks!